build_integration_test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+cli="./faas-cli"
+if [ $(uname) == 'Darwin' ] ; then
+    cli="./faas-cli-darwin"
+elif [ -f "/etc/debian_version" ] ; then
+    if [ $(dpkg --print-architecture) == 'armhf' ] ; then
+        cli="./faas-cli-armhf"
+    elif [ $(dpkg --print-architecture) == 'arm64' ] ; then
+        cli="./faas-cli-arm64"
+    fi
+fi
+
+eval $cli new $FUNCTION --lang python3
+
+cat << EOF > $FUNCTION/handler.py
+def handle(req):
+
+    return "Function output from integration testing: Hello World!"
+EOF
+
+eval $cli build -f $FUNCTION.yml
+
+id=$(docker run --env fprocess="python3 index.py" --name $FUNCTION -p 38080:8080 -d $FUNCTION:latest)
+
+curl -s 127.0.0.1:38080 > got.txt
+
+cat << EOF > want.txt
+Function output from integration testing: Hello World!
+EOF
+
+if cmp got.txt want.txt ; then
```

Comment:
Wonder if this should use the `[ ]` syntax? @ewilde can you review please as the resident bash expert?

Reply:
@alexellis I agree more consistent to stick with the [ ] syntax see: https://ryanstutorials.net/bash-scripting-tutorial/bash-if-statements.php#test